src/sql/workbench/browser/designer/designer.ts

```diff
@@ -50,11 +52,12 @@ export interface IDesignerStyle {
 	selectBoxStyles?: ISelectBoxStyles;
 	checkboxStyles?: ICheckboxStyles;
 	buttonStyles?: IButtonStyles;
+	dropdownStyles?: IListStyles & IInputBoxStyles & IDropdownStyles;
```

Comment:
the dropdown component should have a definition for its own style so that you don't have to know the details here. please create a new interface for dropdown styles if it is not already there.

Reply:
this is the definition of dropdown styles